site/content/docs/developing/domain.en.md

```diff
@@ -78,12 +78,9 @@ Under the hood, Copilot
 <iframe width="560" height="315" src="https://www.youtube.com/embed/Oyr-n59mVjI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ### Use domain in your existing validated certificates
-If you'd like more granular control over the generated ACM certificate, or if the [default `alias` options](#customized-domain-alias) aren't flexible enough, you can specify the `--import-cert-arns` flag when creating the environment to import validated ACM certificates that include the alias. For example:
-
-```console
-$ copilot env init --import-cert-arns arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
-```
-Alternatively, you can always add the certificate later by modifying the [environment manifest](../manifest/environment.en.md):
+If you'd like more granular control over the generated ACM certificate, or if the [default `alias` options](#customized-domain-alias) aren't flexible enough,
+you can import to your environments validated ACM certificates that include the alias.
```

Comment:
```suggestion
you can import a validated ACM certificate that includes the alias to your environment.
```

Reply:
you can import multiple certificates, so I addressed the fb but used plural instead "you can import validated ACM certificates that include the alias to your environment."